src/components/ExitMovie/ExitMovie.js

```diff
@@ -0,0 +1,8 @@
+import './ExitMovie.css';
```

Comment:
nothing in this file - same comment as above

Reply:
yeah, it's the same file as above, I'll remove the import